README.md

```diff
@@ -507,6 +509,526 @@ source scripts/makeAllPlots.sh figures/ResponseUniformityMaps
 
 this will then execute genericPlotter taking each `*.cfg` file in the `figures/ResponseUniformityMaps` directory.
 
+### 3.b.ii.I Helper Script - Parsing Excel Files to genericPlotter 
+
+- **Main script:** Produce\_Config\_File.py
+- **Additional scripts:** QC3.py, QC4\_HV.py, QC4\_SS.py, QC5\_Gain.py, QC5\_Rate.py
+
+
+**Main script:** This tool is designed to read selected columns from an excel file and produce a config file in the genericPlotter input style described in session: [4.e.iv. Plot Config File](#4eiv-plot-config-file). Following, the produced config file can be used as an input file by genericPlotter to create a graph in the official CMS style.
+
+The following **options** are defined:
+
+Field Name | Type | Description
+---------- | ---- | -----------
+`-h, --help` |  | All the options that are available are displayed.
+`-f, --file` |list of strings  | Add an entry to the input excel file list.
+`--OutputName` |string | Set Output file name, default is: Myconfig.
+`--SelectSheetNum` | int| Select the Excel sheet for reading, default is: 0 (counting starts from 0).
+`--SelectColumnX` |int | Select Column number for X data, default value is 0 (counting starts from 0).
+`--SelectColumnY` | int| Select Column number for Y data, default value is 0 (counting starts from 0).
+`--SelectRowStart` |int |Select the first Row for reading, default is 1 (counting starts from 0).
+`--SelectRowEnd` | int|  Select the last Row for reading, default is 61 (counting starts from 0, the last row is not included).
```

Comment:
Can you change the algorithm to be more intuitive for the user:

> Select the last Row for reading, default is 61 (counting starts from 0, the last row is not included)

E.g. this should be the last Row, but then the user is told the last row is not included.  So for ease of use make this really be the last row, e.g. if you pass `Row X` to the code and the code won't use X, have the code take `Row X+1` so this is easier for the user.  Then the last row they give is really the last row.

Reply:
done..

Reply:
done